package.json

```
@@ -17,38 +17,44 @@
   ],
   "repository": "nuxt-community/web-vitals-module",
   "license": "MIT",
+  "type": "module",
   "exports": {
-    ".": "./dist/index.js",
-    "./*": "./*",
-    "./dist/runtime/*": "./dist/runtime/*.mjs"
+    ".": {
+      "import": "./dist/module.mjs",
+      "require": "./dist/module.cjs"
+    }
   },
-  "main": "./dist/index.js",
-  "types": "./dist/index.d.ts",
+  "main": "./dist/module.cjs",
+  "types": "./dist/types.d.ts",
   "files": [
     "dist"
   ],
   "scripts": {
-    "build": "siroc build",
-    "dev": "nuxt playground",
-    "lint": "eslint --ext .ts .",
-    "prepublishOnly": "yarn build",
-    "release": "yarn test && standard-version && npm publish && git push --follow-tags",
-    "test": "yarn lint"
+    "prepack": "nuxt-module-build",
+    "dev": "nuxi dev playground",
+    "dev:build": "nuxi build playground",
+    "dev:prepare": "nuxt-module-build --stub && nuxi prepare playground",
+    "docs": "cd docs && yarn dev",
+    "release": "nuxt-module-build && standard-version && git push --follow-tags && npm publish",
+    "lint": "eslint --ext .js,.ts,.vue"
   },
   "dependencies": {
-    "defu": "^5.0.0",
-    "ufo": "^0.7.9",
+    "@nuxt/kit": "3.0.0-rc.4",
```

(please follow https://github.com/nuxt/starter/tree/module)

```suggestion
    "@nuxt/kit": "^3.0.0-rc.4",
```